src/content_scripts/protoots.js

```diff
@@ -105,39 +105,34 @@ function main() {
  */
 async function fetchPronouns(statusID, account_name) {
 	// log(`searching for ${account_name}`);
-
-	let cacheResult = await browser.storage.local.get().then(getSuccess, onError);
+	let cacheResult = { pronounsCache: {} };
+	try {
+		cacheResult = await browser.storage.local.get();
+	} catch {
+		// ignore errors, we have an empty object as fallback.
+	}
 
 	if (account_name[0] == "@") account_name = account_name.substring(1);
```

Comment:
this statement was added to have account_name format match an author's 'acct' field. but in the current version of the code this statement is only used once - for logging - where account_name works just as well
idk why i'm putting it here, just writing it down so i remember to get rid of that